### ISSUE TITLE
Import a folder tree

### ISSUE THREAD

**Issue by noraj** [notable/notable#662]
### Feature description

Joplin has a nice feature: it can import simple markdown file(s) (like notable) but also can import a tree structure (folder contains folders containing foders ... containing markdown notes).

### Feature motivation

It is a pain to import folder by bolder a very classed tree structure of markdown notes, for examples exported by other notes taking apps.

**Comment by fabiospampinato** [comment]
Sounds like a reasonable feature, but I think under Windows there's no API for selecting either a folder or a file, so we should have to provide 2 separate menu entries for this. I'm not sure how Joplin does it, I'll investigate.

**Comment by noraj** [comment]
Fore Joplin see here https://github.com/laurent22/joplin/blob/897f53b13eb7ade7c99f8748cff3d5ff3fc8858b/ReactNativeClient/lib/services/InteropService.js and here https://github.com/laurent22/joplin/blob/897f53b13eb7ade7c99f8748cff3d5ff3fc8858b/ReactNativeClient/lib/services/InteropService_Importer_Md.js
There is no reason Notable can't handle it the same way. Where is the problem for Windows?

**Comment by fabiospampinato** [comment]
@noraj Aren't those snippets about react native? Anyway I can't read joplin's source right now. I've taken a look at Electron's docs and it seems they provide an API for selecting both files and folders, maybe I'm misremembering something 🤔

**Comment by noraj** [comment]
@fabiospampinato You're right, since tehre is a `ElectronClient` fodler, there is maybe an equivalent for Electron inside.

> I've taken a look at Electron's docs about this and it seems they provide an API for selecting both files and folders, maybe I'm misremembering something

I can't tell you I don't know a thing about JS.

**Comment by noraj** [comment]
Even VScode has an menu entry called "Open Folder", it may help.

**Comment by Ivansstyle** [comment]
Hi! I had high hopes for this app, but there folder structure import is a feature that is lacking. I store my notes in markdown, and currently use shell / vscode to access them. I found for myself, and many others a folder tree structure is best way to store these notes. This way, it is possible to store general notes at a base, and detailed stuff deeper in the structure. 
Also, I have an issue using notable without folders, because I use git to sync all my notes. And I have work notes that I sync with my work git server and personal that I keep on personal server. I cannot use a single folder for both. 
Please add this feature! Thank you
